### PR TITLE
Docs: Typo improved: Correctness grammar

### PR DESCRIPTION
**Pull request by pmbaldha**
The noun phrase <b>forked repository</b> seems to be missing a determiner before it. Consider adding an article.

### CODE REVIEW

**Comment by talldan** [comment]
I think there are lots of little improvements that could be made here. What do you think about updating the PR to include more fixes @pmbaldha?

I think it could be updated to the following:

- Fork the Gutenberg repository.
- Clone the forked repository.
- Create a new branch.
- Make code changes.
- Commit the code changes within the newly created branch.
- Push the branch to the forked repository.
- Submit a Pull Request to the Gutenberg repository.

**Comment by pmbaldha** [comment]
@talldan @Soean I have just made changes as @talldan suggested. Can you please review it? Thank you very much

File: docs/contributors/git-workflow.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-- Commit code changes within the newly created branch
-- Push branch to forked repository
-- Submit Pull Request to Gutenberg repository
+- Fork the Gutenberg repository.
+- Clone the forked repository.
+- Create a new branch.
+- Make code changes.
+- Commit the code changes within the newly created branch.
+- Push the branch to forked repository.
+- Submit a Pull Request to Gutenberg repository.
```
Comment (marekhrabe) [inline docs/contributors/git-workflow.md]: I think these two are missing:

```suggestion
- Push the branch to the forked repository.
- Submit a Pull Request to the Gutenberg repository.
```

**Comment by pmbaldha** [comment]
@marekhrabe I have committed your changes.

**Review by marekhrabe — APPROVED** [review]
Thank you!

**Comment by paaljoachim** [comment]
It would be very helpful to get a Quick Guide. A step by step terminal command approach similar to this:
https://developer.wordpress.org/block-editor/tutorials/devenv/#quickstart

Taking the list @talldan mentioned expanding it with the terminal commands used.

Thank you!

File: docs/contributors/git-workflow.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-- Commit code changes within the newly created branch
-- Push branch to forked repository
-- Submit Pull Request to Gutenberg repository
+- Fork the Gutenberg repository.
+- Clone the forked repository.
+- Create a new branch.
+- Make code changes.
+- Commit the code changes within the newly created branch.
+- Push the branch to the forked repository.
+- Submit a Pull Request to the Gutenberg repository.
```
Comment (paaljoachim) [inline docs/contributors/git-workflow.md:11]: Can we here add a Quick Guide approach? Using the above list and adding in the terminal commands used,

**Comment by talldan** [comment]
@paaljoachim This PR was just fixing some typos to the existing documentation (the list of steps was already present), so I don't think it's fair to ask this contributor to make the changes you're requesting.

Would you be able to make a separate issue for any improvements?

**Comment by paaljoachim** [comment]
Hey Dan. I totally agree on your comment.
I am not sure which documentation this is in relation to, so that it why I added the comment here. I can of course add a new issue where I link to the correct place in the doc (I just need the link). Thanks.

**Comment by talldan** [comment]
@paaljoachim Here's the docs:
https://github.com/WordPress/gutenberg/blob/master/docs/contributors/git-workflow.md

Looks like there already detailed instructions after the summary, so no further changes needed.

**Pull request merged**